Guidelines.md

```diff
@@ -627,7 +627,7 @@ Add an Access-Control-Max-Age pref response header containing the number of seco
 Because browser preflight response caches are notoriously weak, the additional round trip from a preflight response hurts performance.
 Services used by interactive Web clients where performance is critical SHOULD avoid patterns that cause a preflight request
 - For GET and HEAD calls, avoid requiring request headers that are not part of the simple set above. Allow them to be provided as query parameters instead.
-  - The Authorization header is not part of the simple set, so the authentication token MUST be sent through the "access_token" query parameter instead, for resources requiring authentication. Note that passing authentication tokens in the URL is not recommended, because it can lead to the token getting recorded in server logs and exposed to anyone with access to those logs. Services that accept authentication tokens through the URL MUST take steps to mitigate the security risks, such as using short-lived authentication tokens, suppressing the auth token from getting logged, and controlling access to server logs.
+  - The Authorization header (AH) SHOULD be used as method of providing authentication information (thus, can be included in the simple set). AH provides authentication of the IP header and next-level protocol data. Security services are initiated between two communicating hosts, between two communicating security gateways or between a security gateway and a host. Note that passing authentication tokens in the URL is not recommended, because it can lead to the token getting recorded in server logs and exposed to anyone with access to those logs. Services that accept authentication tokens through the URL MUST take steps to mitigate the security risks, such as using short-lived authentication tokens, suppressing the auth token from getting logged, and controlling access to server logs.
```

Comment:
I think just a minor modification to the last part of the paragraph is what is needed.  e.g.

> Although the Authorization Header is not part of the CORS simple set, passing authentication tokens in the URL is not recommended, because it can lead to the token getting recorded in server logs and exposed to anyone with access to those logs. Services that accept authentication tokens through the URL MUST take steps to mitigate the security risks, such as using short-lived authentication tokens, suppressing the auth token from getting logged, and controlling access to server logs. 

I don't think we need to start trying to explain what the Authorization Header is, or how it can be used.

Reply:
Thank you for the feedback. Will make changes now.